### PR TITLE
Add security overview to github-security-features.md

### PR DESCRIPTION
**Pull request by lucie-docs**
### Why:

Closes https://github.com/github/docs-content/issues/6198 

### What's being changed:

Added the security overview to the "Available with GitHub Advanced Security" section for GHEC and GHES 3.2 and above.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by lucie-docs** [comment]
I'm not sure how to solve the check not passing, I did confirm that the issue linked in the PR is fine to have there 😞

**Comment by lucie-docs** [comment]
Never mind, posting the previous comment solved the check haha

**Comment by ramyaparimi** [comment]
@lucie-docs 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/code-security/getting-started/github-security-features.md [hunk 1]
```diff
@@ -84,6 +84,12 @@ Automatically detect tokens or credentials that have been checked into a reposit
 Show the full impact of changes to dependencies and see details of any vulnerable versions before you merge a pull request. For more information, see "[About dependency review](/code-security/supply-chain-security/about-dependency-review)."
 {% endif %}
 
+{% ifversion ghec or ghes > 3.1 %}
+### Security overview
+
+Review the security configuration and alerts for an organization and identify the repositories at greatest risk. For more information, see "[About the security overview](/code-security/security-overview/about-the-security-overview)."
```
Comment (mchammer01) [inline content/code-security/getting-started/github-security-features.md:90]: Nit
```suggestion
Review the security configuration and alerts for your organization and identify the repositories at greatest risk. For more information, see "[About the security overview](/code-security/security-overview/about-the-security-overview)."
```

**Review by mchammer01 — DISMISSED** [review]
LGTM ✨ - thanks for adding this omission @lucie-docs

**Review by mchammer01 — APPROVED** [review]
LGTM :shipit:

**Pull request merged**